packages/svelte/src/internal/shared/clone.js

```diff
@@ -75,8 +80,12 @@ function clone(value, cloned, path, paths) {
 			const copy = {};
 			cloned.set(value, copy);
 
+			if (original !== null) {
+				cloned.set(original, copy);
+			}
+
 			for (var key in value) {
-				// @ts-expect-error
+				// @ts-ignore
```

Comment:
what was this change for? we want to avoid ts-ignore as far as possible

```suggestion
				// @ts-expect-error
```